.github/workflows/release.yml

```diff
@@ -190,7 +148,7 @@ jobs:
           source env/bin/activate
           pip3 install -q pyyaml dohq-artifactory requests boxsdk tqdm
           
-          echo ${{ env.BUILD_NAME }} ${{ env.BUILD_NUMBER }} ${{ env.BUILD_VERSION }}
+           echo ${{ env.BUILD_NAME }} ${{ env.BUILD_NUMBER }} ${{ env.BUILD_VERSION }}
```

Comment:
extra unnneeded white space

Reply:
removed extra space

Reply:
I still see the extra space just before the `echo`